examples/gateways/artifact-gateway-configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: artifact-gateway-configmap
+  namespace: argo-events
+data:
+  s3.myartifact: |-
```

Comment:
how did you choose this naming convention? I see `s3.myartifact` and others such as `calendar.myConfig` and `nats.myConfig`.. is this standardized across all gateways?

Reply:
They are just random names I came up to test gateways. I haven't thought of how naming convention should like. Also do we need a convention? coz when user creates a custom gateway, its upto he/she to interpret the configuration.